net/instaweb/rewriter/cache_extender_test.cc

```diff
@@ -1157,6 +1157,44 @@ TEST_F(CacheExtenderTest, VaryOrigin) {
                    StringPrintf(kCssFormat, cache_extended_css.c_str()));
 }
 
+TEST_F(CacheExtenderTest, RenderCsp) {
+  InitTest(100);
```

Comment:
It's the TTL (in seconds) the fixture will give the resources. It mostly doesn't matter for this test (some of the others use AdvanceTime, etc), just has to be in a range that we would cache extend. Could probably use the existing 'kShortTtlSec ' constant, though.